f8a_worker/dispatcher/flows/componentApiFlow.yml

```diff
@@ -0,0 +1,8 @@
+---
+  flow-definitions:
+    - name: 'componentApiFlow'
```

Comment:
@fridex here are the references -

https://github.com/samuzzal-choudhury/fabric8-analytics-server/blob/90b4e167cc1dbddccaf040591bb228d483ed1ccd/bayesian/utils.py#L35

https://github.com/samuzzal-choudhury/fabric8-analytics-server/blob/90b4e167cc1dbddccaf040591bb228d483ed1ccd/bayesian/api_v1.py#L266

https://github.com/samuzzal-choudhury/fabric8-analytics-server/blob/90b4e167cc1dbddccaf040591bb228d483ed1ccd/bayesian/utils.py#L59